docs/fundamentals/database-collection.txt

```diff
@@ -0,0 +1,176 @@
+.. _laravel-db-coll:
+
+=========================
+Databases and Collections
+=========================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: php framework, odm
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use {+odm-short+} to access
+and manage MongoDB databases and collections.
+
+MongoDB organizes data in a hierarchical structure. A MongoDB
+deployment contains one or more **databases**, and each database
+contains one or more **collections**. In each collection, MongoDB stores
+data as **documents** that contain field-and-value pairs. In
+{+odm-short+}, you can access documents through Eloquent models.
+
+To learn more about the document data format,
+see :manual:`Documents </core/document/>` in the Server manual.
+
+.. _laravel-access-db:
+
+Specify the Database in a Connection Configuration
+--------------------------------------------------
+
+You specify a database name when configuring a connection in your
+application's ``config/database.php`` file. The ``connections`` property
```

Comment:
Suggestion:

I think this sentence specifies a time ("when"), but I don't think it's connected to  any sequence of instructions we provide. I think the following sentence could avoid associating it with an order:

You can configure the name of the database that a connection uses in your application's config/database.php configuration file.